.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,46 +1,39 @@
 <!-- This is a pull request template, you do not need to uncomment or remove the comments, they won't show up in the PR text. -->
-<!-- Your Pull Request name should start with one of the following tags -->
-<!-- [NEW] For new features -->
-<!-- [FIX] For bug fixes -->
-<!-- [BREAK] For pull requests including breaking changes -->
 
-## Proposed changes
-<!-- Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue below. -->
+<!-- Your Pull Request name should start with one of the following tags
+  [NEW] For new features
+  [IMPROVE] For a improvement (performance or little improvements) in existent features
+  [FIX] For bug fixes that affects the end user
+  [BREAK] For pull requests including breaking changes
+  Chore: For small tasks
+  Doc: For documentation
+-->
+
+<!-- Checklist!!! If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code. 
+  - I have read the [CONTRIBUTING](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md#contributing-to-rocketchat) doc
+  - I have signed the [CLA](https://cla-assistant.io/RocketChat/Rocket.Chat)
```

Comment:
since this will not be render as markdown, does it makes sense to have them as markdown links?
```suggestion
  - I have read the Contributing Guide - https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md#contributing-to-rocketchat doc
  - I have signed the CLA - https://cla-assistant.io/RocketChat/Rocket.Chat
```